src/avfoundation.cs

```diff
@@ -3363,8 +3366,7 @@ interface AVFragmentedAssetTrack {
 
 	interface IAVCaptureFileOutputDelegate {}
 
-#if MONOMAC 
-
+	[NoWatch][NoiOS][NoTV]
```

Comment:
Does this one need a Mac Attribute?

Reply:
Or since we are in a "mac-friendly" file then the generator will just slap one on here correct?

Reply:
By default we'll get the older supported version.